### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by iv1310**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,43 @@ PolkaVerse is a niche social site built on the Subsocial network. It focuses on
 
 Visit Subsocial's [website](https://subsocial.network) to learn more about the project.
 
+## Guide to build and deploy polkaverse 
+
+1. Prepare the dockerfile and adjust the config if needed.
+2. Build the image in local with this command and please ensure to add build argument.
+```bash
+$ docker build --build-arg GH_GA_ID=valueREDACTED --build-arg GH_APP_KIND=valueREDACTED --build-arg GH_HCAPTCHA_SITE_KEY=valueREDACTED --build-arg GH_AMP_ID=valueREDACTED --build-arg GH_OFFCHAIN_SIGNER_URL=valueREDACTED --build-arg GH_CONNECTION_KIND=valueREDACTED --build-arg GH_SELLER_CLIENT_ID=valueREDACTED --build-arg GH_SERVER_MNEMONIC==valueREDACTED --build-arg GH_SELLER_TOKEN_SIGNER=valueREDACTED --build-arg GH_NEXT_PUBLIC_DATAHUB_QUERY_URL=valueREDACTED --build-arg GH_NEXT_PUBLIC_DATAHUB_SUBSCRIPTION_URL=valueREDACTED --build-arg GH_DATAHUB_QUEUE_URL=valueREDACTED --build-arg GH_DATAHUB_QUEUE_TOKEN=valueREDACTED -t polkaverse-docker-image:v1.0.0 .
```
Comment (oap75) [inline README.md:14]: Let's use latest tag instead of versioning.

File: README.md [hunk 2]
```diff
@@ -6,6 +6,43 @@ PolkaVerse is a niche social site built on the Subsocial network. It focuses on
 
 Visit Subsocial's [website](https://subsocial.network) to learn more about the project.
 
+## Guide to build and deploy polkaverse 
+
+1. Prepare the dockerfile and adjust the config if needed.
```
Comment (oap75) [inline README.md:11]: Mention the existing Dockerfile.

File: README.md [hunk 3]
```diff
@@ -6,6 +6,43 @@ PolkaVerse is a niche social site built on the Subsocial network. It focuses on
...
+2. Build the image in local with this command and please ensure to add build argument.
+```bash
+$ docker build --build-arg GH_GA_ID=valueREDACTED --build-arg GH_APP_KIND=valueREDACTED --build-arg GH_HCAPTCHA_SITE_KEY=valueREDACTED --build-arg GH_AMP_ID=valueREDACTED --build-arg GH_OFFCHAIN_SIGNER_URL=valueREDACTED --build-arg GH_CONNECTION_KIND=valueREDACTED --build-arg GH_SELLER_CLIENT_ID=valueREDACTED --build-arg GH_SERVER_MNEMONIC==valueREDACTED --build-arg GH_SELLER_TOKEN_SIGNER=valueREDACTED --build-arg GH_NEXT_PUBLIC_DATAHUB_QUERY_URL=valueREDACTED --build-arg GH_NEXT_PUBLIC_DATAHUB_SUBSCRIPTION_URL=valueREDACTED --build-arg GH_DATAHUB_QUEUE_URL=valueREDACTED --build-arg GH_DATAHUB_QUEUE_TOKEN=valueREDACTED -t polkaverse-docker-image:v1.0.0 .
+```
+Notes:
+Please execute the build process with theses build arguments, you need to specify the value.
+* [x] GH_GA_ID=valueREDACTED
```
Comment (oap75) [inline README.md:18]: Remove check box and use `*` .

File: README.md [hunk 4]
```diff
@@ -6,6 +6,43 @@ PolkaVerse is a niche social site built on the Subsocial network. It focuses on
...
+* [x] GH_SELLER_CLIENT_ID=valueREDACTED
+* [x] GH_SELLER_TOKEN_SIGNER=valueREDACTED
+* [x] GH_SERVER_MNEMONIC=valueREDACTED
+* [x] GH_NEXT_PUBLIC_DATAHUB_QUERY_URL=valueREDACTED
+* [x] GH_NEXT_PUBLIC_DATAHUB_SUBSCRIPTION_URL=valueREDACTED
+* [x] GH_DATAHUB_QUEUE_URL=valueREDACTED
+* [x] GH_DATAHUB_QUEUE_TOKEN=valueREDACTED
+3. Then check the docker images that has been builded.
+```bash
+$ docker images
```
Comment (oap75) [inline README.md:33]: grep based on polkaverse.

File: README.md [hunk 5]
```diff
@@ -6,6 +6,43 @@ PolkaVerse is a niche social site built on the Subsocial network. It focuses on
...
+```
+4. To run the docker images with docker-compose, please prepare the docker-compose.yaml config file at first.
```
Comment (oap75) [inline README.md:35]: Create the `docker-compose.yaml` file and refer to it. 

Let's create just a simple compose file with polkaverse and expose it to the localhost.

File: README.md [hunk 6]
```diff
@@ -6,6 +6,43 @@ PolkaVerse is a niche social site built on the Subsocial network. It focuses on
...
+```bash
+$ docker-compose -f docker-compose.yaml up -d
+```
+5. Check the running container with this command.
+```bash
+$ docker-compose ps
+$ docker-compose logs
+```
+
```
Comment (oap75) [inline README.md:44]: Add another step to reach their local app. like `http://locahost:3003`

**Review by oap75 — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (iv1310) [reply README.md:64]: Done ya

File: README.md [hunk 7]
```diff
@@ -6,6 +6,63 @@ PolkaVerse is a niche social site built on the Subsocial network. It focuses on
...
+1. To run the docker images with docker-compose, please prepare the docker-compose.yaml config file at first.
+```yaml
+# docker-compose.yml
+version: "3"
+services:
+  web-ui:
+    image: polkaverse-docker-image:latest
+    ports:
+      - "3002:3002"
+      - "3003:3003"
```
Comment (oap75) [inline README.md:47]: It's almost ready. Just define what are these ports for.

**Review by oap75 — CHANGES_REQUESTED** [review]

**Review by oap75 — APPROVED** [review]
Just fixup commits. LGTM

**Pull request merged**